docker-compose/technical/grafana/dashboards/pods_supervision.json

```diff
@@ -1126,13 +1126,213 @@
       ],
       "type": "table"
     },
+    {
```

Comment:
I'm not sure these gateway metrics fits in "pods supervision"? In my opinion, it should be in a separate dashboard.